app/src/components/inputs/inputUserSearch/inputUserSearch.jsx

```diff
@@ -37,15 +36,15 @@ const newOptionCreator = (option) => ({
 });
 const promptTextCreator = (label) => label;
 const makeURL = (input, isAdmin, projectId) =>
-  !isAdmin ? URLS.projectUserSearchUser(projectId, input) : URLS.searchUsers(input);
+  isAdmin ? URLS.searchUsers(input) : URLS.projectUserSearchUser(projectId, input);
 const makeOptions = (options, projectId) =>
   options.map((option) => ({
     userName: option.fullName || '',
     userLogin: option.userId,
     email: option.email || '',
     disabled: !!option.assignedProjects[projectId],
     isAssigned: !!option.assignedProjects[projectId],
-    userAvatar: getPhoto(option.userId),
+    userAvatar: URLS.dataUserPhotoByProject(projectId, option.userId, true),
```

Comment:
I don't see this function un `urls.js`. Did you mean `dataUserPhoto`?

Reply:
Oh, sorry, missed it. Thank you